ext/featurefmt/rpm/contentmanifest/updater.go

```diff
@@ -0,0 +1,170 @@
+package contentmanifest
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/quay/clair/v3/pkg/httputil"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	lastUpdateDate  = time.Date(1970, time.January, 1, 1, 1, 1, 1, time.UTC)
```

Comment:
Can't this just be the zero value of a time.Time?
https://play.golang.org/p/nxRHJPuN_mR

Reply:
Fixed